cypress/integration/emoji/recently_used_emoji_spec.js

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/* eslint max-nested-callbacks: ["error", 3] */
+
+import {getRandomInt} from '../../utils';
+
+describe('Recent Emoji', () => {
+    before(() => {
+        // 1. Login and go to /
+        cy.login('user-1');
+        cy.visit('/');
+    });
+
+    it('M14014 Recently used emojis are shown 1st', async () => {
+        const maxRandom = 400;
+
+        // 2. Get random emoji index
+        const firstEmoji = getRandomInt(maxRandom);
+
+        // 3. Show emoji list
+        cy.get('#emojiPickerButton').should('be.visible').click();
+
+        // 4. Click emoji with random index
+        cy.get('.emoji-picker__item').eq(firstEmoji).click();
+
+        // 5. Submit post
+        cy.get('#create_post').submit();
+
+        // 6. Open emoji list
+        cy.get('#emojiPickerButton').click();
+
+        // 7. Get second emoji
+        const secondEmoji = getRandomInt(maxRandom);
+
```

Comment:
Second emoji is not doing reaction to a post.  I think you missed to add `cy.clickPostReactionIcon();` here.

Reply:
did you mean submit post ?

Reply:
Nope, I mean react to the last post.

When I ran the test, the second emoji (`:man_dancing:`) is just drafted in the post_texbox like in the screenshot:
![Screen Shot 2019-03-26 at 9 03 29 PM](https://user-images.githubusercontent.com/5334504/54999345-fca3d080-500a-11e9-95db-9027d887049a.png)

I was hoping that the second emoji is a reaction to the last post like:
![Screen Shot 2019-03-26 at 9 08 14 PM](https://user-images.githubusercontent.com/5334504/54999479-4b516a80-500b-11e9-96d0-ab775d067c3d.png)

Reply:
Oh I see, wait a second